Test/Microsoft.ApplicationInsights.Test/Shared/Extensibility/MetricManagerTest.cs

```diff
@@ -20,10 +20,10 @@ public void CanCreateMetricHavingNoDimensions()
             var sentTelemetry = new List<ITelemetry>();
 
             var client = this.InitializeTelemetryClient(sentTelemetry);
-            using (MetricManager manager = new MetricManager(client))
+            using (MetricManagerV1 manager = new MetricManagerV1(client))
```

Comment:
You are right, it would have been better to do this as separate PRs.Unfortunately I discovered the need for this renaming in the process of integrating and they ended up embedded. I can redo those changes in a separate branch for a separate PR, but it will take a little time. Given that they concern a separate set of files I hope that it does not affect the PR logic too much. Can we please skip this improvement?

Reply:
Have you tried opening and reviewing this PR? It hangs chrome on my machine =)

It literally takes a few minutes to merge as there will be only handful conflicts. I just tried. And will save a lot of time and nerves to reviewers. So please consider doing it